src/app/pages/account/signup/page.module.css

```diff
@@ -0,0 +1,154 @@
+.container{
```

Comment:
it would be good that the name of your css file page.module.css you change it for a more meaningful name so that there is no confusion.